.storybook/main.ts

```diff
@@ -0,0 +1,47 @@
+import type {StorybookConfig} from '@storybook/react-vite';
+import {mergeConfig} from 'vite';
+import path from 'path';
+
+const config: StorybookConfig = {
+  stories: [
+    '../packages/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+    '../performance-benchmarks/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+  ],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-a11y',
+    '@storybook/addon-interactions',
+    './addons/google-analytics/register',
+  ],
+  framework: {
+    name: '@storybook/react-vite',
+    options: {legacyRootApi: true},
+  },
+  features: {
+    interactionsDebugger: true,
+  },
+  staticDirs: ['./static'],
+  typescript: {
+    // enable type checking
+    check: true,
+  },
+  docs: {
+    docsPage: 'automatic',
+    defaultName: 'Docs',
+  },
+  async viteFinal(config) {
```

Comment:
Question: does the profiler still work without the aliasing and subsequently forcing a dev build for production?

Reply:
Will verify that.

Reply:
Had to change `process.env` to `import.meta.env` (Vite's version). Also needed to [include the aliases](https://github.com/twilio-labs/paste/pull/2895/commits/d3bce166f50b04a1c067931978c20e56455801b2) plus one for `react-dom/client` which may help inform the React 18 upgrade. Verified this gets profiling to work as expected.